### ISSUE TITLE
Suggestion: Consider changing default color from ColorWhite to ColorClear

### ISSUE THREAD

**Issue by DannyBen** [gizak/termui#249]
Right now, when working with light terminal theme, there are quite a few places that need to be overridden from `ColorWhite` to `ColorClear` (or some other color that is readable on white background).

Not only this makes development and experimentation a little lengthier, I suspect it also causes developers who publish packages that rely on termui to have a default theme that is not friendly to light background users.

I believe that if `ColorClear` is used as the default color, and both `ColorBlack` and `ColorWhite` are avoided in all the defaults, this will be a good starting point for everybody, and will provide a sensible theme that works out of the box for both dark and light background users.

**Comment by drgrib** [comment]
Agreed on this. I was going mad trying to get it to change from `ColorWhite` to the default terminal color and had no idea it was `ColorClear` until I saw this post. It shouldn't default to `ColorWhite`. This presupposes a very specific system setup and ignores the colors users have personally selected for readability in their own terminals.
